src/core.js

```diff
@@ -10,6 +10,7 @@ import config from "./core/config";
 import { defined, extend, objectType, is, now } from "./core/utilities";
 import { registerLoggingCallbacks, runLoggingCallbacks } from "./core/logging";
 import { sourceFromStacktrace } from "./core/stacktrace";
+import {} from "./core/onerror";
```

Comment:
You be able to just do:

```
import "./core/onerror";
```

Similar to what is done in `src/qunit.js`. I prefer this pattern as it is a good indicator that you simply want the module evaluated, but don't expect exports from it.